docs/architecture/README.md

```diff
@@ -29,71 +29,73 @@ Note the context/background should be written in the present tense.
 
 ## Table of Contents
 
-### Accepted
+### Implemented
```

Comment:
So for the status we currently have the option of: Deprecated | Proposed | Accepted | Declined. I would expect these options to be reflected in the table of contents thus listing the ADR's by status. 

I think given that the time between being accepted and implemented can be a long time that we should draw the distinction in the status. I also find the status `Proposed` to be quite redundant i.e. the PR that lands the ADR should be where a proposal is decided if it is accepted or not. Hence I would propose that status should be one of: Deprecated | Declined | Approved | Implemented

Reply:
Agree with the proposal, ill make the changes